src/index.css

```diff
@@ -1,3 +1,277 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@100;300;400;500;600;700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Slab:wght@100;300;500;700&display=swap');
+
 body {
   box-sizing: border-box;
+  padding: 4%;
+  margin: 0;
+  font-family: "Montserrat", sans-serif;
+  background-color: #f3f4f8;
+}
+
+#root {
+  background-color: #feffff;
+}
+
+header {
+  display: flex;
+  padding: 24px 4%;
+  justify-content: space-between;
+  border-bottom: 1px solid #e8e8e8;
+}
+
+nav {
+  display: flex;
```

Comment:
- [ ] Its good to use reusable classes instead of applying the same CSS property over and over, kindly do so for `display: flex` in all instances where it re-occurs in this file

Reply:
Thanks for the suggestion. Fixed🛠️✅